Documentation/configuration/api-rate-limiting.rst

```diff
@@ -0,0 +1,135 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _api_rate_limiting:
+
+*****************
+API Rate Limiting
+*****************
+
+The per node Cilium agent is essentially event-driven. For example, the CNI
+plugin is invoked when a new workload is scheduled onto the node which in turn
+makes an API call to the Cilium agent to allocate an IP address and create the
+Cilium endpoint. Another example is loading of network policy of services
+definitions where changes of these definitions will create and event which will
+notify the Cilium agent that a modification is required.
+
+Due to being event-driven, the amount of work performed by the Cilium agent
+highly depends on the rate of external events it receives. In order to
+constrain the resources that the Cilium agent consumes, it can be helpful to
+restrict the rate and allowed parallel executions of API calls.
+
+Default Rate Limits
+===================
+
+The following API calls are currently subject to rate limiting:
+
+========================== ====== ===== ============= ============ ================= =========== ===============================
+API Call                   Limit  Burst Max Parallel  Min Parallel Max Wait Duration Auto Adjust Estimated Processing Duration
+========================== ====== ===== ============= ============ ================= =========== ===============================
+``PUT /endpoint/{id}``     0.5/s  4     4                          15s               True        2s
+``DELETE /endpoint/{id}``               4             4                              True        200ms
+``GET /endpoint/{id}/*``   4/s    4     4             2            10s               True        200ms
+``PATCH /endpoint/{id}*``  0.5/s  4     4                          15s               True        1s
```

Comment:
Nit: missing forward slash?
```suggestion
``PATCH /endpoint/{id}/*``  0.5/s  4     4                          15s               True        1s
```

Reply:
It's intentional, it also covers `PATCH /endpoint/{id}`